kevm-pyk/src/kevm_pyk/kproj/evm-semantics/driver.md

```diff
@@ -510,9 +521,13 @@ Here we check the other post-conditions associated with an EVM test.
          <blockNonce>       HN </blockNonce>
          <baseFee>          HF </baseFee>
          <withdrawalsRoot>  WR </withdrawalsRoot>
-      requires #blockHeaderHash(HP, HO, HC, HR, HT, HE, HB, HD, HI, HL, HG, HS, HX, HM, HN)         ==Int #asWord(HASH)
-        orBool #blockHeaderHash(HP, HO, HC, HR, HT, HE, HB, HD, HI, HL, HG, HS, HX, HM, HN, HF)     ==Int #asWord(HASH)
-        orBool #blockHeaderHash(HP, HO, HC, HR, HT, HE, HB, HD, HI, HL, HG, HS, HX, HM, HN, HF, WR) ==Int #asWord(HASH)
+         <blobGasUsed>      UB </blobGasUsed>
+         <excessBlobGas>    EB </excessBlobGas>
+         <beaconRoot>       BR </beaconRoot>
```

Comment:
Can I ask what are these  new`blobGasUsed`, `excessBlobGas` and `beaconRoot` cells for?

Reply:
These fields are part of the [EIP-4788](https://eips.ethereum.org/EIPS/eip-4788) and [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844). They have yet to be used as the EIPs still need to be implemented. They must be defined in the configuration to parse and decode the RLP of the Ethereum-test test suite correctly.